cmd/crc/cmd/config/view.go

```diff
@@ -55,6 +60,9 @@ func runConfigView(cfg map[string]config.SettingValue, tmpl *template.Template,
 		if v.IsDefault {
 			continue
 		}
+		if v.IsSecret && !showSecrets {
+			continue
+		}
```

Comment:
The previous commit could possibly have `if v.IsSecret { continue }`, and this commit would only add the `&& !showSecret` part of the test.